server/data/data-jpa/src/main/java/org/apache/james/mailrepository/jpa/MimeMessageJPASource.java

```diff
@@ -0,0 +1,35 @@
+package org.apache.james.mailrepository.jpa;
```

Comment:
We are missing the compulsory ASF V2 header license.

Reply:
My bad.